src/generate-js.js

```diff
@@ -1,11 +1,28 @@
 /**
- * All functions to generate js files
+ * generateJSFile - Generates all the javascript
  *
- * generateJSFile - TODO
- **/
+ * @param  {type} pkg - The package object with blender key
+ * @return {type}     - A result object with a js key
+ */
+const path = require('path');
+const fs = require('fs');
+
+const { color } = require('./color.js');
+const { D, log } = require('./log.js');
 
 function generateJSFile(pkg) {
-	return pkg;
+	const pkgPath = path.normalize(`${pkg.path}/${pkg.pkg.js}`);
+
+	let js = null;
+
+	try {
+		js = fs.readFileSync(pkgPath);
+		D.log(`Found file at "${color.yellow(pkgPath)}"`);
```

Comment:
We can't log. We have to return so that the API can handle the error them self differently than the CLI